common.php

```diff
@@ -51,31 +51,31 @@
 	'DIGESTS_DAILY'						=> 'Journalier',
 	'DIGESTS_DATE'						=> 'Date',
 	'DIGESTS_DELIMITER'					=> ' ‹ ', // Utilisé pour faciliter l'affichage de la hiérarchie des noms de forum depuis l'index jusqu'en bas. Ne devrait pas changer dans les traductions linguistiques, sauf si un caractère différent est utilisé dans le fil d'Ariane.
-	'DIGESTS_DISABLED_MESSAGE'			=> 'Pour activer les champs, sélectionnez Réglages généraux et sélectionnez le type de résumé',
+	'DIGESTS_DISABLED_MESSAGE'			=> 'Pour accéder aux options de cette page, accédez au menu « Options de base » puis sélectionnez un type de résumé',
 		'DIGESTS_DISCLAIMER_HTML'			=> '<p><a href="%5$s">Désabonnement en un clic</a></p><p>Vous pouvez aussi modifier ou supprimer votre abonnement aux forums depuis le <a href="%1$sucp.%3$s">Panneau de contrôle de l’utilisateur</a>.</p><p>Si vous avez des questions ou si vous souhaitez nous contacter veuillez adresser votre courriel au <a href="mailto:%4$s?subject=Digests">webmaster des forums de %2$s</a>.</p>',
 	'DIGESTS_DISCLAIMER_TEXT'			=> 'Désabonnement en un clic : %5$s
 	
 Vous pouvez aussi modifier ou supprimer votre abonnement aux forums depuis le Panneau de contrôle de l’utilisateur.
 	
 Si vous avez des questions ou si vous souhaitez nous contacter veuillez adresser votre courriel au webmaster des forums : %4$s.',
-	'DIGESTS_EXPLANATION'				=> 'Les résumés qui sont envoyés périodiquement sont constitués à partir des messages réalisés sur le forum. Les résumés peuvent être envoyés à une récurence journalière, hebdomadaire ou mensuelle et à une heure que vous choisisisez. Vous pouvez spécifier les forums dont les messages vous intéressent plus particulièrement, ou par défaut, vous pouvez sélectionner choisir de recevoir tous les messages de tous les forums. Vous pouvez annuler votre abonnement à tout moment en revenant sur cette page. Nombreux sont les utilisateurs qui trouvent ces résumés très utiles. Nous vous encourageons à faire un essai !',
+	'DIGESTS_EXPLANATION'				=> 'Les résumés envoyés périodiquement sont constitués à partir des messages réalisés sur le forum. Ils peuvent être envoyés quotidiennement, hebdomadairement ou mensuellement, à l’heure de votre choix. Vous pouvez spécifier les forums dont les messages vous intéressent plus particulièrement ou, par défaut, vous pouvez choisir de recevoir tous les messages de tous les forums. Vous pouvez annuler votre abonnement à tout moment en revenant sur cette page. Nombreux sont les utilisateurs qui trouvent ces résumés très utiles. Nous vous encourageons à faire un essai !',
```

Comment:
`à partir des messages réalisés sur le forum.`  
`réalisés` me perturbe 🤔   

La phrase anglaise est la suivante : 
`Digests are email summaries of posts that are sent to you periodically.`

`Les résumés sont un condensé des messages publiés qui vous sont envoyés périodiquement par courriel. `  
ou sinon, en gardant le plus de termes de la traduction FR  
`Les résumés sont constitués des messages publiés sur le forum et vous sont envoyés périodiquement par courriel. `

Reply:
Les résumés récapitulent les messages publiés sur le forum et vous sont envoyés périodiquement par courriel. Ils peuvent être expédiés quotidiennement, hebdomadairement ou mensuellement, à l’heure de votre choix.

Reply:
Parfait !